util/sdk/macros/index.mjs

```diff
@@ -73,11 +75,32 @@ const isPublicEventMethod = and(
   compose(
     option(false),
     map(_ => true),
-      chain(find(propEq('name', 'event'))),
+    chain(
+      find(
+        and(
+          propEq('name', 'event'),
+          propSatisfies('x-provides', isEmpty)
+        )
+      )
+    ),
     getPath(['tags'])
   )
 )
 
+const isProviderMethod = compose(
```

Comment:
Seems like you're also doing this in `util/docs/macros/index.mjs`. Is determining if a method is a provider method or not done differently in the docs macros?

Reply:
It is the same, I don't know why sdk uses crocks and docs doesn't, but I just kept it in the same pattern as what it was already doing with events

Reply:
cause my small brain was incapable of following @cletustboone's example of crocking & highlighting at the same time 😂